site/src/pages/CreateTokenPage/CreateTokenForm.tsx

```diff
@@ -21,6 +22,8 @@ import {
   customLifetimeDay,
 } from "./utils";
 
+dayjs.extend(utc);
```

Comment:
I've seen Storybook catch so many of these "depending on a dayjs plugin that we don't actually install locally" bugs. 🐛